.github/workflows/bcftools_bgzip.yml

```diff
@@ -0,0 +1,40 @@
+name: bcftools_bgzip
```

Comment:
You can delete all of these files now and just add the relevant entries for each module to `.github/filters.yml` instead.